### PR TITLE
INSTALL.md: add gentoo

### PR DESCRIPTION
**Pull request by anbj**
Add gentoo to INSTALL.md.

### CODE REVIEW

File: INSTALL.md
Comment (meghanrjones) [inline INSTALL.md:194]:
```suggestion
**NOTE:** This may provide old GMT versions. Consider [building from source](BUILDING.md).
```

**Comment by meghanrjones** [comment]
Looks good to me, though I have never used gentoo so I cannot confirm that it works.

**Comment by anbj** [comment]
I have tested it. It works, but installs a hopelessly old version:

```
$ gmt --version
5.4.4
```

However, I think it's worth showing that GMT *is* available on Gentoo.

**Review by meghanrjones — APPROVED** [review]

**Comment by meghanrjones** [comment]
@anbj, you should be able to squash and merge this PR when you have time.

**Comment by anbj** [comment]
Got it.

**Comment by anbj** [comment]
I've added a section about WSL. Please review. Suggestions are welcome.

File: doc/rst/source/cookbook/non-unix-platforms.rst
Comment (meghanrjones) [inline doc/rst/source/cookbook/non-unix-platforms.rst:41]:
```suggestion
See `Windows Subsystem for Linux Documentation <https://docs.microsoft.com/en-us/windows/wsl/>`_ to get started.
```

File: doc/rst/source/cookbook/non-unix-platforms.rst
Comment (meghanrjones) [inline doc/rst/source/cookbook/non-unix-platforms.rst:43]:
```suggestion
Once WSL is installed, choose a distribution (e.g. Debian) and follow the `install <https://github.com/GenericMappingTools/gmt/blob/master/INSTALL.md#ubuntudebian>`_
```

File: doc/rst/source/cookbook/non-unix-platforms.rst
Comment (meghanrjones) [inline doc/rst/source/cookbook/non-unix-platforms.rst:44]: Underscores are needed after the links to make them render properly in sphinx.
```suggestion
instructions or `build from source <https://github.com/GenericMappingTools/gmt/blob/master/BUILDING.md>`_.
```

**Review by meghanrjones — COMMENTED** [review]
Thanks for adding the section! Looks good to me, I just have a few formatting suggestions. @joa-quim might want to check it out before it gets merged.

**Comment by joa-quim** [comment]
Just add a note that WSL is only available on Win Pro version(s)

**Comment by anbj** [comment]
Are you sure Joaquim? I thought WSL was available regardless of this (https://docs.microsoft.com/en-us/windows/wsl/wsl2-faq).

**Comment by joa-quim** [comment]
OK, if they say so. But I remember that some two years ago having tried in a Win 10 Home and WSL (1 at that time) was not available.

**Comment by anbj** [comment]
Yes, that might have been the case.

**Pull request merged**
